test/tests/functional/pbs_cgroups_hook.py

```diff
@@ -4539,18 +4547,20 @@ def test_cgroup_cgswap(self, vnode_per_numa_node=False):
             self.logger.info('First Mom has no swap, test will just '
                              'check if job cgswap is added')
             a = {'Resource_List.select':
-                 '1:ncpus=1:mem=100mb:vmem=1100mb:vnode=%s'
+                 '1:ncpus=0:mem=100mb:vmem=1100mb:vnode=%s'
```

Comment:
It is foolish to _assume_ that your test harness has two cpus if you don't need to assume it.

If you don't do this, then you would in theory need to add a skip test so that if resources_available.ncpus is less than 2 you skip part of the test -- cumbersome.

Here we're only testing memory, so we can get away with asking for zero cpus (which will simply put job tasks in /sys/fs/cgroup/cpuset/pbs_jobs.service/jobid by default and give them a low number of shares).